packages/api/src/routes/medical/document.ts

```diff
@@ -90,15 +95,17 @@ router.post(
     const patientId = getFromQueryOrFail("patientId", req);
     const facilityId = getFromQueryOrFail("facilityId", req);
     const override = stringToBoolean(getFrom("query").optional("override", req));
+    const requestId = uuidv7();
```

Comment:
Routes shouldn't be defining domain stuff. You can see how this is a problem considering the fact that the logic to define the `requestId` is now spread across diff files, not to mention the callers/clients of the domain/app need to do something extra when calling `queryDocumentsAcrossHIEs`.